### PR TITLE
add TAPI formulae

### PR DESCRIPTION
**Pull request by Torrekie**
as issue #52 required

### CODE REVIEW

**Comment by Gcenx** [comment]
Thanks for the PR let me do add some notes

File: Formula/tapi.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class Tapi < Formula
+  desc "Apple TAPI library (tpoechtrager fork)"
+  homepage "https://opensource.apple.com/tarballs/tapi"
+  url "https://github.com/tpoechtrager/apple-libtapi/archive/1100.0.11.tar.gz"
+  sha256 "bef360e713852f451383a7f1fdd4abfd17ee2d937778cd757708e52595d77b78"
+  license "APSL-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
```
Comment (Gcenx) [inline Formula/tapi.rb:9]: This shouldn’t be required I’ve build this against stock macOS make before

File: Formula/tapi.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  patch :p0, :DATA
```
Comment (Gcenx) [inline Formula/tapi.rb:11]: Honestly don’t need the documentation so might as well skip the patch

File: Formula/tapi.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  def install
+    ENV.cxx11
```
Comment (Gcenx) [inline Formula/tapi.rb:11]: This shouldn’t be required, I’m guessing you based this of the Macports Portfile I’d done a while ago. In that we set that for the compiler standard to ensure a new enough compiler would be use for ancient systems.

**Comment by Gcenx** [comment]
I believe this should also be keg only as Xcode includes libtapi, we’re unable to use the Xcode provided copy due to the lack of headers.

File: Formula/tapi.rb (hunk 3 above)
Comment (Torrekie) [reply Formula/tapi.rb:11]: then removing this should be okay

**Comment by Torrekie** [comment]
I haven't seen corresponding xcrun stub executable of `tapi` in `/usr/bin` and CLT's binary path was not automatically exported to PATH

File: Formula/tapi.rb (hunk 3 above)
Comment (Gcenx) [reply Formula/tapi.rb:11]: I believe it should be needed that setting should be handled within CMakeLists.txt so nothing for us to worry about.

Comment (Gcenx) [reply Formula/tapi.rb:11]: *Should not

**Comment by Gcenx** [comment]
> I haven't seen corresponding xcrun stub executable of `tapi` in `/usr/bin` and CLT's binary path was not automatically exported to PATH

That’s true but I’m still not sure having libtapi symlinked into /usr/local/lib is a good idea. For example cctools would also endup being keg-only or it could cause issues being in $PATH

**Comment by Gcenx** [comment]
Being in a standard library search path

**Comment by Gcenx** [comment]
With any luck brew testbot is happy then I’ll merge

**Comment by Gcenx** [comment]
Thanks @Torrekie

**Pull request closed without merging**